recipes/gmp/all/conandata.yml

```diff
@@ -1,7 +1,7 @@
 sources:
   "6.2.1":
-    url: "https://gmplib.org/download/gmp/gmp-6.2.1.tar.bz2"
-    sha256: "eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c7a36b4d7c"
+    url: "https://github.com/alisw/GMP/archive/refs/tags/v6.2.1.tar.gz"
+    sha256: "fb776e19e0267053c3d6ffef0dcc001fb441da0e2b5224cfd9656aa2f1b8614d"
```

Comment:
I am not a fan of the checksum changing, why are they different?

Reply:
Because the original file is `.tar.bz2` and the new one is `.tar.gz`

Reply:
Another option is https://ftp.gnu.org/gnu/gmp/gmp-6.2.1.tar.xz but also will change the checksum

Reply:
https://ftp.gnu.org/gnu/gmp/gmp-6.2.1.tar.bz2 is the right option